src/store/getters/state-getters.ts

```diff
@@ -513,7 +513,8 @@ export const isIncorporationApplicationValid = (state: StateIF): boolean => {
 /** Whether all the registration steps are valid. */
 export const isRegistrationValid = (state: StateIF): boolean => {
   const isCertifyValid = getCertifyState(state).valid && !!getCertifyState(state).certifiedBy
-  const isFeeAcknowledgementValid = getRegistration(state).feeAcknowledgement
+  // const isFeeAcknowledgementValid = getRegistration(state).feeAcknowledgement
+  const isFeeAcknowledgementValid = true // FUTURE: use line above instead
```

Comment:
This caused the review page to be invalid, because the component and data are coded but not displayed (so the user can never check the checkbox).